src/scsi/scsi.c

```diff
@@ -60,7 +61,7 @@ static SCSI_CARD scsi_cards[] = {
     { "[ISA] Adaptec AHA-154xB",	"aha154xb",	&aha154xb_device,	},
     { "[ISA] Adaptec AHA-154xC",	"aha154xc",	&aha154xc_device,	},
     { "[ISA] Adaptec AHA-154xCF",	"aha154xcf",	&aha154xcf_device,	},
-	{ "[ISA] BusLogic BT-542B",		"bt542b",	&buslogic_542b_1991_device,	},
+    { "[ISA] BusLogic BT-542B",		"bt542b",	&buslogic_542b_1991_device,	},
```

Comment:
indenting

Reply:
I've corrected that indenting to match the others.

Reply:
oh sorry, didn't notice. nevermind :P